docs/faqs.md

```diff
@@ -260,3 +261,70 @@ In addition, we support only 64-bit builds of the interpreter (amd64/x86_64 arch
 
 There are no guarantees that recipes will work correctly in future Python versions having breaking changes to the interpreter,
  as we don't test all the possible combinations (and probably will never be). Patches are welcomed if problems are found.
+
+## How to package libraries that depend on Intel MKL, IPP or DNN libraries?
+
+There are several popular software libraries provided by Intel:
+
+* [Intel Math Kernel Library (MKL)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)
+* [Intel Integrated Performance Primitives (IPP)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/ipp.html)
+* [Intel Deep Neural Networking Library (DNN)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onednn.html)
+
+these Intel libraries are widely used by the various well-known open-source projects (e.g. [OpenCV](https://opencv.org/) or [TensorFlow](https://www.tensorflow.org/)).
+
+unfortunately, these Intel libraries cannot be accepted into ConanCenter due to the several important reasons:
+
+* they are closed-source and commercial products, which we're not going to re-distribute via ConanCenter due to the license restrictions
+* registration on the Intel portal is required in order to dowload the libraries, there are no permanent public direct download links
+* they use graphical installers which are hard to automate within conan recipe
+* they are pre-built binaries, so they will never respect any custom compiler flags specified
+
+instead, the libraries that depend on *MKL*, *IPP* or *DNN* should use the following references:
+
+* `intel-mkl/<version>`, e.g. `intel-mkl/2021`
+* `intel-ipp/<version>`, e.g. `intel-ipp/2021`
+* `intel-dnn/<version>`, e.g. `intel-dnn/2021`
+
+**NOTE**: These references are not availabe in ConanCenter and will be never available! it's consumer's responsibility to provide the recipes for that libraries.
+
+for instance, the following skeleton could be used as a starting point (given Intel MKL as an example):
+
+```
+from conans import ConanFile, tools
+
+class ConanMKL(ConanFile):
+    name = "intel-mkl"
+
+    def package_info(self):
+        self.cpp_info.includedirs = ["/opt/intel/mkl/include"]
+        self.cpp_info.libdirs = ["/opt/intel/mkl/lib"]
+```
+
+(most likely, the real recipe will use much more sophisticated logic to locate an existing installation, e.g. via environment variables)
```

Comment:
Not sure if I would comment about this here. I think we should add a dedicated entry in the docs about _special recipes_, a place where we explain motivation and skeleton for some of them: system recipes, proxy ones, these ones,...

```suggestion
```

Reply:
yeah, it's very simple anyway, and we can add it to the `examples` repository, if needed